src/index/middleware/tokenValidator.ts

```diff
@@ -58,10 +59,17 @@ export function createMiddleware(app: Orizuru.IServer, provider: string, opts?:
 			const accessToken = extractAccessToken(req, tokenRegex);
 			const userInfo = await validateAccessToken(accessToken, opts) as OpenIDTokenWithStandardClaims;
 
-			const user = {
+			let user: User | SalesforceUser = {
```

Comment:
OPTIONAL: Or you may think this is a bit weird...

```suggestion
			const user: User | SalesforceUser = {
				username: userInfo.preferred_username,
				...userInfo.organization_id && { organizationId: userInfo.organization_id }
			};

```

Reply:
Changed to avoid the situation of setting user twice (which is what this was trying to stop)